src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -135,6 +136,8 @@ def pluralize(word, nb):
             ))
             if 'analyzer' in part:
                 analyzers.append(part['analyzer'])
+            elif 'analyzer_custom' in part:
+                analyzers.append(part['analyzer_custom'].format(files=' '.join(cls_stats['publishable_paths'])))
```

Comment:
Nit: I would limit the maximum number of paths displayed to avoid building a huge comments when a large number of defect is analyzed. We can ship without this, and may patch it if we see too many big comments

Reply:
Yes I would like to see how this behaves first, since the main intent of this is to have an actionable message.